roles/k8s_best_practices_certsuite/tasks/pre-run.yml

```diff
@@ -238,8 +238,8 @@
     name: redhatci.ocp.mirror_images
   vars:
     mi_images:
-      - "quay.io/testnetworkfunction/cnf-test-partner:latest"
```

Comment:
I would change the task for this (we don't need to do this for the certsuite-sample-workload image)

```
- name: "Save images in local registry"
  ansible.builtin.include_role:
    name: redhatci.ocp.mirror_images
  vars:
    _kbpc_repo_org_name: "{{ (kbpc_version is version('v5.2.3', '>=') or kbpc_version == 'HEAD') | ternary('redhat-best-practices-for-k8s', 'testnetworkfunction') }}" # noqa: redhat-ci[no-role-prefix]
    mi_images:
      - "quay.io/redhat-best-practices-for-k8s/certsuite-sample-workload:latest"
      - "quay.io/{{ _kbpc_repo_org_name }}/{{ kbpc_support_image_name }}:{{ kbpc_support_image_version }}"
    mi_authfile: "{{ kbpc_pullsecret }}"
    mi_registry: "{{ kbpc_registry }}"
  when:
    - kbpc_pullsecret | length
    - kbpc_registry | length
```